var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -51,7 +51,9 @@ class Flex(AutotoolsPackage):
 
     def url_for_version(self, version):
         url = "https://github.com/westes/flex"
-        if version >= Version('2.6.1'):
+        if version >= Version('2.6.3'):
+            url += "/releases/download/v{0}/flex-{0}.tar.gz".format(version)
+        elif version >= Version('2.6.1'):
```

Comment:
The whole `url_for_version()` can and should be simplified.  In my tests, ALL versions can be obtained from the "archives" folder.  Please get rid of `url_for_version()` entirely and set `url='https://github.com/westes/flex/archive/v2.6.3.tar.gz'`.  That should work; @adamjstewart is there an easy way to test that all versions can download and checksum with the new URL?